src/layouts/SplitContentLayout/SecondaryContent/styles/SidebarList/Item.js

```diff
@@ -27,7 +27,7 @@ export default class SidebarListItem extends React.PureComponent {
     /**
      * A component for rendering a container of an item
      */
-    Container: PropTypes.func,
+    Container: PropTypes.oneOfType([PropTypes.object, PropTypes.func]),
```

Comment:
I think it should be `PropTypes.oneOfType([PropTypes.Node, PropTypes.func])`

Reply:
These are Component classes being passed in, not rendered nodes. That's why `func` was used originally.

Reply:
// A React element.
  optionalElement: PropTypes.element,

then `element` should work